src/errorhandler.cpp

```diff
@@ -38,7 +38,7 @@ ErrorHandler::ErrorType ErrorHandler::toErrorType(
     case QNetworkReply::HostNotFoundError:
       [[fallthrough]];
     case QNetworkReply::TimeoutError:
```

Comment:
This is wrong. Host-Not-Found should not be treated at "VPNDepedentConnectionError' right?

Reply:
Right, that was not supposed to be changed.